src/SelectSingle.vue

```diff
@@ -187,7 +197,6 @@
 			selectOption(index: number) {
 				const selected = this.options[index]
 				this.inputValue = selected.label
-				this.selectedIndex = index
```

Comment:
We shouldn't update selectedIndex manually, because selectedIndex depends on selected value and we should recalculate selectedIndex only when value of select was changed. Old approach doesn't handle a case when value is updated by code.